pkgs/development/python-modules/skops/default.nix

```diff
@@ -10,6 +10,7 @@
   matplotlib,
   pandas,
   scikit-learn,
+  stdenvNoCC,
```

Comment:
nah, we already pull in stdenv via buildPythonPackage
```suggestion
```

Reply:
Done. I learn something new with each PR :)

Reply:
Got it

Reply:
Apparently it won't parse withut the explicit import. (ofBorg just failed it for that reason).